www/views/preferencesCashShuffle.html

```diff
@@ -1,35 +1,34 @@
 <ion-view id="tab-cspreferences" class="settings" show-tabs>
   <ion-nav-bar>
-    <ion-nav-title>{{'CashShuffle' | translate}}</ion-nav-title>
+    <ion-nav-title>{{'Privacy Mode' | translate}}</ion-nav-title>
     <ion-nav-back-button>
     </ion-nav-back-button>
   </ion-nav-bar>
 
   <ion-content>
+
     <div class="list">
 
+      <div class="comment" translate>
+        <img src="../img/obfuscate.svg" alt="CashShuffle" class="cs-img">
+        <div class="settings-description-cs" translate>
+          <p>CashShuffle is a fully decentralized coin mixing protocol that shuffles your Bitcoin Cash with other network participants. This shuffling process obscures your real spending and makes it difficult for chain analysis companies to follow your transactions.</p>
+            <p><strong>Warning: Privacy mode is in beta.</strong> Shuffling your wallet can take hours to complete. Do not enable if you need to spend right away.
+            </p>
+            <div class="cs-button">
+            <a href="https://www.youtube.com/watch?v=8g0Levpj1I4" class="button-green" target="_blank">Learn more</a>
```

Comment:
Link does not function on iOS. Will need to use external link module.

Reply:
@maplesyrupsucker